.github/actions/components/caching/action.yml

```diff
@@ -22,6 +22,8 @@ runs:
           **/tfplan
           **/swagger.yaml
           **/infrastructure/terraform/**/output.json
+          .downloads
```

Comment:
This relates to all the downloady bits, im sure we didnt do it this way with NRLF so wondering whats different this time for pulling down the JAR file? The only thing I can think of is that we are doing this on the fly in the CI?

Reply:
i just needed somewhere to download the JAR file to :D don't mind where it goes, but somewhere at the repo root that is gitignored. What do you suggest?